[2025-08-14T03:41:54.848+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-14T03:41:54.927+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: playstore_review_etl_pipeline.validate_raw_reviews manual__2025-08-14T03:41:16.307492+00:00 [queued]>
[2025-08-14T03:41:54.948+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: playstore_review_etl_pipeline.validate_raw_reviews manual__2025-08-14T03:41:16.307492+00:00 [queued]>
[2025-08-14T03:41:54.951+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-08-14T03:41:54.981+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): validate_raw_reviews> on 2025-08-14 03:41:16.307492+00:00
[2025-08-14T03:41:55.006+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'playstore_review_etl_pipeline', 'validate_raw_reviews', 'manual__2025-08-14T03:41:16.307492+00:00', '--job-id', '117', '--raw', '--subdir', 'DAGS_FOLDER/my_pipeline.py', '--cfg-path', '/tmp/tmpj_50fpro']
[2025-08-14T03:41:55.014+0000] {standard_task_runner.py:91} INFO - Job 117: Subtask validate_raw_reviews
[2025-08-14T03:41:55.014+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=188) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-08-14T03:41:55.016+0000] {standard_task_runner.py:63} INFO - Started process 190 to run task
[2025-08-14T03:41:55.113+0000] {task_command.py:426} INFO - Running <TaskInstance: playstore_review_etl_pipeline.validate_raw_reviews manual__2025-08-14T03:41:16.307492+00:00 [running]> on host 53b91e1bad0a
[2025-08-14T03:41:55.358+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='your_email@domain.com' AIRFLOW_CTX_DAG_OWNER='data_engineering_team' AIRFLOW_CTX_DAG_ID='playstore_review_etl_pipeline' AIRFLOW_CTX_TASK_ID='validate_raw_reviews' AIRFLOW_CTX_EXECUTION_DATE='2025-08-14T03:41:16.307492+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-14T03:41:16.307492+00:00'
[2025-08-14T03:41:55.361+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-14T03:41:55.400+0000] {my_pipeline.py:107} INFO - Running validation on RawAppReviews data (Placeholder).
[2025-08-14T03:41:55.402+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-08-14T03:41:55.404+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-14T03:41:55.424+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=playstore_review_etl_pipeline, task_id=validate_raw_reviews, run_id=manual__2025-08-14T03:41:16.307492+00:00, execution_date=20250814T034116, start_date=20250814T034154, end_date=20250814T034155
[2025-08-14T03:41:55.477+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-14T03:41:55.547+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-08-14T03:41:55.554+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
